packages/components/Organisms/Accordion/Accordion.vue

```diff
@@ -31,17 +28,14 @@
             <rpl-icon symbol="arrow_down_tertiary" color="primary" class="rpl-accordion__icon" :class="{'rpl-accordion__icon--expanded': accordionIsOpen(index)}"/>
           </button>
         </h2>
-        <transition
-          name="accordion"
-          @enter="start"
-          @after-enter="end"
-          @before-leave="start"
-          @after-leave="end"
```

Comment:
Hey @rubyllamadora good point. It's because I am using height to do the accordion expend/collapse transition. Which has to be fixed number. When we do window resize, the fixed height need to be updated to avoid the scroll bar. I will have a look! Thanks!

Reply:
Hey @rubyllamadora please review again. I have fixed the window resize height issue.